### PR TITLE
docs: Update README.md

### LINKED ISSUE #23: Make changes to README.md
Changes are required as done in PR #22

### PR DESCRIPTION
**Pull request by yash1412**
fixes #23

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,91 +65,110 @@ Okay, now you know why to document let's define some rules that everyone has to
...
+
 1. Format of the letter sent to all the previously requested schools/universities for workshops
 2. The package proposed to be held - The course covered for 2 days/3 days (Mechanical, Electrical/Electronics, and Computer); Kit taken along; the per head charge taken
 3. A complete report of the conducted workshop - Place, Institute held in, attendance of students, how many club members invited, total revenue generated(show by subtracting the total travel & accommodation cost)
 4. Any upcoming/incomplete request or possible places/institutes/colleges who showed any amount of interest for conduction of workshop but couldn't be followed up because of preoccupations, awkward timing, non-negotiable fees or finish of tenure - mention all the conversations and outcomes
 
-##### Social Media Coordinator
+### Social Media Coordinator
+
 [Role](https://github.com/auvzhcet/Documentation/blob/master/Social_Media_Manager.md)
```
Comment (pnijhara) [inline README.md]: Change this link.

File: README.md [hunk 2]
```diff
@@ -54,91 +65,110 @@ Okay, now you know why to document let's define some rules that everyone has to
...
+### PCB designer
+
+### Acoustic Engineer
 
 [Comparative Study of hydrophones used by other teams](https://github.com/auvzhcet/AUV2k19/blob/master/hydrophones_study.md)
 
 [Our implemetation](https://github.com/auvzhcet/Documentation/blob/master/Acoustics.md)
 
 ---
-## Computer :computer:
```
Comment (pnijhara) [inline README.md:92]: We can use emojis for readme.
Emojis were removed for index.md only as they are not rendered by readthedocs. README is meant for github only.

File: README.md [hunk 3]
```diff
@@ -54,91 +65,110 @@ Okay, now you know why to document let's define some rules that everyone has to
...
 5. Handle queries from any person trying to know about the content
 6. Handle content from non-club members (Club alumni/experts)
 
+## Mechanical
+
+### Fabrication Engineer
+
+1. Location of acquisition of materials - Each material, exact location, cost with date.
 
-## Mechanical :gear:
```
Comment (pnijhara) [inline README.md:72]: No need to remove emojis.

File: README.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# Documentation :memo:
```
Comment (pnijhara) [inline README.md]: This also.

**Review by pnijhara — CHANGES_REQUESTED** [review]

**Comment by pnijhara** [comment]
Semantic PR check is failing as you have not written the PR title properly. Make it "docs: Update README.md"

File: README.md (hunk 1 above)
Comment (yash1412) [reply README.md]: can you please give me the required link?

Comment (pnijhara) [reply README.md]: Look where `social_media_manager.md` file is.

Comment (yash1412) [reply README.md:170]: I got the file but no link is provided there

Comment (yash1412) [reply README.md]: ok

File: README.md [hunk 5]
```diff
@@ -167,7 +167,7 @@ The following document has been modified on 18th October 2018
 
 ### Social Media Coordinator
 
-[Role](https://github.com/auvzhcet/Documentation/blob/master/Social_Media_Manager.md)
+[Role](https://github.com/auvzhcet/Documentation/blob/master/docs/marketing/Social_Media_Manager.md)
```
Comment (pnijhara) [inline README.md]: Again! This is the github link.
This should be [Role] (docs/marketing/Social_Media_Manager.md)

Make this change to other references also.

**Review by pnijhara — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -54,91 +65,110 @@ Okay, now you know why to document let's define some rules that everyone has to
...
 
-##### PCB designer
-##### Acoustic Engineer
+### PCB designer
+
+### Acoustic Engineer
 
 [Comparative Study of hydrophones used by other teams](https://github.com/auvzhcet/AUV2k19/blob/master/hydrophones_study.md)
 
 [Our implemetation](https://github.com/auvzhcet/Documentation/blob/master/Acoustics.md)
```
Comment (pnijhara) [inline README.md]: Change this link also.

File: README.md [hunk 7]
```diff
@@ -54,91 +65,110 @@ Okay, now you know why to document let's define some rules that everyone has to
...
 The following document has been modified on 18th October 2018
 [Doc given by Abbas bhai](https://github.com/auvzhcet/Documentation/blob/master/BMS%20for%20lithium%20polymer%20battery%20pack.docx)
 
-##### PCB designer
-##### Acoustic Engineer
+### PCB designer
+
+### Acoustic Engineer
 
 [Comparative Study of hydrophones used by other teams](https://github.com/auvzhcet/AUV2k19/blob/master/hydrophones_study.md)
```
Comment (pnijhara) [inline README.md:102]: Change this link.

**Review by pnijhara — CHANGES_REQUESTED** [review]

File: README.md [hunk 8]
```diff
@@ -101,7 +101,7 @@ The following document has been modified on 18th October 2018
...
 
-[Our implemetation](https://github.com/auvzhcet/Documentation/blob/master/Acoustics.md)
+[Our implemetation](docs/computer/Acoustics.md)
```
Comment (pnijhara) [inline README.md]: Spelling mistake!

File: README.md (hunk 7 above)
Comment (pnijhara) [reply README.md:102]: You haven't changed the link!

**Review by pnijhara — CHANGES_REQUESTED** [review]

File: README.md (hunk 8 above)
Comment (yash1412) [reply README.md]: I think it is correct as i have seen in preview and it is working

Comment (pnijhara) [reply README.md]: "Implementation" it is

Comment (yash1412) [reply README.md]: ok

File: README.md (hunk 7 above)
Comment (yash1412) [reply README.md:102]: I'm not able to find this file

**Review by pnijhara — APPROVED** [review]

**Comment by pnijhara** [comment]
Thank you for the contribution @yash1412
Always write proper commit messages.

**Pull request merged**